tests/NuGet.Services.AzureSearch.FunctionalTests/Relevancy/V3RelevancyFunctionalTests.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace NuGet.Services.AzureSearch.FunctionalTests
```

Comment:
These seems like a decent first guess at which cases to test but when things are more stable we should formalized our method for picking these test cases and have an XML doc per case.